bazel/emscripten_deps.bzl

```diff
@@ -98,113 +100,85 @@ filegroup(
 )
 """
 
-def emscripten_deps(emscripten_version = "latest"):
-    version = emscripten_version
+def _emscripten_deps_impl(ctx):
+    version = None
+
+    for mod in ctx.modules:
+        for config in mod.tags.config:
+            if config.version and version != None:
+                fail("More than one emscripten version specified!")
```

Comment:
Not sure about this one. We could also pick the maximum version that was specified.